### ISSUE TITLE
Parallel calculation

### ISSUE THREAD

**Issue by SkyQuant** [ygrenier/SwephNet#3]
Thank you **ygrenier** for nice framework!

Does async and parallel (TPL) computation concurrence with Swiss/JPL/Moshier ephemeris available? Is it thread-safe?

**Comment by ygrenier** [comment]
Thanks.

No this Library is not thread-safe, and don't support async. It's the same for the ygrenier/SwissEphNet.

Unfortunally I don't have the time to work on this Library for the moment. But it could be a good idea to thinking about supports some async methods.

I mark this issue with label "enhancement" to get it on mind.

**Comment by SkyQuant** [comment]
Yeah, it could be useful feature when you calculate time series as well as matrices - two dimensional arrays and lists.
